packages/expo-updates/ios/Tests/CodeSigningSignatureHeaderInfoTests.swift

```diff
@@ -0,0 +1,35 @@
+//  Copyright (c) 2020 650 Industries, Inc. All rights reserved.
+
+import XCTest
+
+@testable import EXUpdates
+
+class CodeSigningSignatureHeaderInfoTests : XCTestCase {
```

Comment:
```suggestion
class CodeSigningSignatureHeaderInfoTests: XCTestCase {
```

The body of the class should also be re-indented 😅 
Besides, could you use `ExpoModulesTestCore` and Quick + Nimble for tests? See tests in `ExpoModulesCore`. I think it's much more convenient way than `XCTest`.

Reply:
I'll add some sort of linting in a follow-up PR so there's no need to review style manually. As for the tests, will also look into it in a follow-up.

For now these are just renames, so to limit the scope of this PR I won't adjust style.

Reply:
We actually have SwiftLint set up, but not integrated with Xcode (I think it's just not convenient). My plan was to add this into the code reviewer CI job and even tried that here: #15639 but then it turned out that GitHub rate-limits API tokens when you add too many comments (surprisingly it's sent in only one request).
I'm going to come back to this PR once I'm done with other more urgent stuff.